starfish/pipeline/pipelinecomponent.py

```diff
@@ -7,6 +8,9 @@ class PipelineComponentType(type):
     This is the metaclass for PipelineComponent.  As each subclass that is _not_ PipelineComponent is created, it sets
     up a map between the algorithm name and the class that implements it.
     """
+
+    _algorithm_to_class_map: Optional[Mapping[str, Type]] = None
```

Comment:
this triggers an error for me.  removing it makes mypy happy.

Reply:
for me, removing it makes the _python_ tests die. I fixed it with `Optional[]` which passes both.